Sources/Build/Command.compile(ClangModule).swift

```diff
@@ -46,13 +47,13 @@ struct ClangModuleBuildMetadata {
     let module: ClangModule
 
     /// Path to working directiory.
-    let prefix: String
+    let prefix: AbsolutePath
 
     /// Extra arguments to append to basic compile and link args.
     let otherArgs: [String]
 
     /// Path to build directory for this module.
-    var buildDirectory: String { return Path.join(prefix, "\(module.c99name).build") }
+    var buildDirectory: AbsolutePath { return prefix.appending(module.c99name + ".build") }
```

Comment:
Agreed.  What do you think should be the behavior of trying to append an empty string or a string containing a `/` in that case?  Throwing an error?